pkg/metrics/features/metrics.go

```diff
@@ -511,6 +512,13 @@ func NewMetrics(withDefaults bool) Metrics {
 				}(),
 			},
 		}),
+
+		ACLBCiliumNodeConfigEnabled: metric.NewGauge(metric.GaugeOpts{
+			Help:      "Cilium Node Config enabled on the agent",
```

Comment:
I am not familiar with dynamic node config, is it important to mention here in the help?

Reply:
I tried on not to explain what each feature is. If users don't know what the feature is they can search it on our docs. I just searched for "Cilium Node Config" in our docs and I got this result https://docs.cilium.io/en/stable/configuration/per-node-config/#ciliumnodeconfig-objects.